style.css

```diff
@@ -0,0 +1,329 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    color:var(--default-black);
+    margin: 0;
+    padding: 0;
```

Comment:
I think it's redundant here as you have already specified the same thing using the universal selector above. This could be avoided here.